content-services/latest/using/content/tags-categories.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Tags and Categories
+---
+
+You can apply tags and categories to your content in the Digital Workspace. Tags and categories help organize your content so that related information is easy to find, whether it be connected through different projects or concepts. For more information on tags and categories and how to set this up using the Control Center, see [tags]({% link content-services/latest/admin/control-center.md %}#tags) or [categories]({% link content-services/latest/admin/control-center.md %}#categories).
+
+## Tags
+
+You can assign tags to your content within the Digital Workspace.
+
+1. Sign into the Digital Workspace.
+
+2. Find the content you want to work with and select it by clicking once on the **Type** icon next to its **Name**.
+
+3. Click the **View Details** button on the top right.
+
+4. Click the **Edit icon** at the bottom of the **Details pane**.
+
+5. Click the **+** symbol next to **Tags**.
+
+6. Search for the tag or tags you want to add to your content, and then click **Save Changes**.
```

Comment:
```suggestion
6. You can either search for existing tags and select them or provide a name and create a new tag (it will be created in Control Center and applied to the content as well). After applying all tags click **Save Changes**.
```